RemoteIDModule/status.cpp

```diff
@@ -215,16 +221,16 @@ String status_json(void)
         { "SYSTEM:AreaFloor", String(UAS_data.System.AreaFloor) },
         { "SYSTEM:CategoryEU", ENUM_MAP(cateu, UAS_data.System.CategoryEU) },
         { "SYSTEM:ClassEU", ENUM_MAP(classeu, UAS_data.System.ClassEU) },
-        { "SYSTEM:OperatorAltitudeGeo", String(UAS_data.System.OperatorAltitudeGeo) },
+        { "SYSTEM:OperatorAltitudeGeo", String(opealtgeo_str) },
         { "SYSTEM:Timestamp", String(UAS_data.System.Timestamp) },
         { "LOCATION:Status", ENUM_MAP(status, UAS_data.Location.Status) },
         { "LOCATION:Direction", String(UAS_data.Location.Direction) },
         { "LOCATION:SpeedHorizontal", String(UAS_data.Location.SpeedHorizontal) },
         { "LOCATION:SpeedVertical", String(UAS_data.Location.SpeedVertical) },
         { "LOCATION:Latitude", String(UAS_data.Location.Latitude, 8) },
         { "LOCATION:Longitude", String(UAS_data.Location.Longitude, 8) },
-        { "LOCATION:AltitudeBaro", String(UAS_data.Location.AltitudeBaro) },
-        { "LOCATION:AltitudeGeo", String(UAS_data.Location.AltitudeGeo) },
+        { "LOCATION:AltitudeBaro", String(altbaro_str) },
+        { "LOCATION:AltitudeGeo", String(altgeo_str) },
         { "LOCATION:HeightType", ENUM_MAP(height, UAS_data.Location.HeightType) },
         { "LOCATION:Height", String(UAS_data.Location.Height) },
```

Comment:
Fix the LOCATION:Height at the same time?

Reply:
@friissoren san
I agree. I will also change the height.